Dockerfile

```
@@ -3,10 +3,14 @@ FROM node:14.15.0-alpine
 WORKDIR /stremio
 
 ARG VERSION=master
+ENV FORCE_HTTPS=0
 
 RUN apk add --no-cache openssl-dev wget ffmpeg
 RUN wget https://dl.strem.io/four/${VERSION}/server.js
 RUN wget https://dl.strem.io/four/${VERSION}/stremio.asar
+ADD start.sh /stremio
```

We're already in workdir `/stremio` so you can just do:

```suggestion
COPY start.sh .
```